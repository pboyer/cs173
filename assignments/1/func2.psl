defvar f = lambda(x, y) { +("Chapter ", x, ": ", y, "\n") } in {
  print(f("One", "Object-Oriented Programming"));
  print(f("Two", "Functional Programming"));
  "";
}