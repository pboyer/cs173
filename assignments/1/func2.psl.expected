Chapter One: Object-Oriented Programming
Chapter Two: Functional Programming

